packages/core/src/modules/employment/use-cases/send-company-review-notifications.ts

```diff
@@ -0,0 +1,63 @@
+import dedent from 'dedent';
+
+import { db } from '@oyster/db';
+
+import { job } from '@/infrastructure/bull/use-cases/job';
+import { ENV } from '@/shared/env';
+
+/**
+ * This is a weekly job that runs and finds all students that have had a work
+ * experience end in the last week. Then, it has the ColorStack bot send a DM
+ * to all these students suggesting that they add a review of their experience.
+ */
+export async function sendCompanyReviewNotifications() {
+  const oneMonthAgo = new Date();
+
+  oneMonthAgo.setDate(oneMonthAgo.getDate() - 32);
```

Comment:
We might have to think about this logic a bit. If we have this running every 32 days, couldn't it send notifications for 2 months? Potentially if this is scheduled correctly this isn't an issue, but it is something to think about to ensure the logic makes sense :)